test/logflare/logs/otel_trace_test.exs

```diff
@@ -9,6 +9,25 @@ defmodule Logflare.Logs.OtelTraceTest do
       %{resource_spans: TestUtilsGrpc.random_resource_span(), source: source}
     end
 
+    test "attributes", %{
+      resource_spans: resource_spans,
+      source: source
+    } do
+      [%{"attributes" => a} | _] = OtelTrace.handle_batch(resource_spans, source)
+      assert a != %{}
+      assert Enum.any?(Map.values(a), &is_list/1)
+      assert Enum.any?(Map.values(a), &is_number/1)
+      assert Enum.any?(Map.values(a), &is_binary/1)
+      assert Enum.any?(Map.values(a), &is_boolean/1)
+    end
+
+    test "array handling", %{
+      resource_spans: resource_spans,
+      source: source
+    } do
+      batch = OtelTrace.handle_batch(resource_spans, source)
+    end
```

Comment:
This doesn't assert anything and `handle_batch` is already called above with the same arguments. I would expect this test to be either expanded or removed :)

Reply:
removed as it array is already covered in above test.